frontend/constants/networks.ts

```diff
@@ -10,14 +11,16 @@ export interface Network {
 const NETWORKS = {
   mainnet: {
     chainId: 42,
+    name: 'LUKSO',
     symbol: 'LYX',
     rpcUrl: 'https://rpc.lukso.gateway.fm',
     explorerURL: 'https://explorer.execution.mainnet.lukso.network',
-    universalGraveForwarder: '0xa5467dfe7019bf2c7c5f7a707711b9d4cad118c8',
-    lsp1UrdVault: '',
+    universalGraveForwarder: '0x433908ce6457b302a6452257Bc40e466d95c78E9',
+    lsp1UrdVault: '0x9292dAf1cdc3d03a1A0BbD4B3319C49A3B91d703',
```

Comment:
Where did you find this?

Reply:
Oh I see, you deployed in solidity/contracts/LSP1UniversalReceiverDelegateVault.sol ?

Reply:
@democrazyfi yes ser :)